src/views/Drive/components/DriveExplorer/components/DriveItemContextMenu.tsx

```diff
@@ -93,14 +94,48 @@ const getDownloadMenuItem = (downloadItems: (target?) => void) => ({
   },
 });
 
-const getVersionHistoryMenuItem = (viewVersionHistory: (target?) => void) => ({
-  name: t('drive.dropdown.versionHistory'),
-  icon: ClockCounterClockwise,
-  action: viewVersionHistory,
-  disabled: (item) => {
-    return item.isFolder;
-  },
-});
+export type VersionHistoryMenuConfig = {
+  locked?: boolean;
+  onLockedClick?: () => void;
+  allowedExtension?: boolean;
```

Comment:
Remember that Booleans should be read as a question (using prefixes such as is, has, can)